tests/neynarAPI.ts

```diff
@@ -0,0 +1,609 @@
+        // eslint-disable-next-line no-console
+        console.log(registeredSigner);
+      });
+      it("can fetch a signer", async function () {
+        const signer = await client.fetchSigner(signerUuid ?? "");
+        expectDefined(signer);
+        expect(signer.public_key).to.be.eq(signerPublicKey);
+      });
+    });
+
+    describe("#v1fetchCast", function () {
+      it("can fetch an existing cast", async function () {
+        const existingCastHash = "0x0cac69b3162e2db93af22eb0156a1ecb6d2641e1";
+        const cast = await client.v1FetchCast(existingCastHash);
+        expectDefined(cast);
+        expect(cast.hash).to.eq(existingCastHash);
+      });
+
+      it("returns undefined for nonexistent cast", async function () {
+        const nonexistentCastHash =
+          "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+        const cast = await client.v1FetchCast(nonexistentCastHash);
+        expect(cast).to.be.undefined;
+      });
+    });
+
+    describe("#fetchCast", function () {
+      it("can fetch an existing cast", async function () {
+        const existingCastHash = "0x0cac69b3162e2db93af22eb0156a1ecb6d2641e1";
+        const cast = await client.fetchCast(existingCastHash);
+        expectDefined(cast);
+        expect(cast.hash).to.eq(existingCastHash);
+      });
+
+      it("returns undefined for nonexistent cast", async function () {
+        const nonexistentCastHash =
+          "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+        const cast = await client.fetchCast(nonexistentCastHash);
+        expect(cast).to.be.undefined;
+      });
+    });
+
+    describe("#fetchCastsInThread", function () {
+      it("can fetch multiple pages of casts", async function () {
+        const threadHash = "0xd02442da75c1a09c0b0a735f9d6fdfb0db287d89";
+        const castsInThread = await client.fetchCastsInThread({
+          hash: threadHash,
+        });
+        expectDefined(castsInThread);
+        for (const cast of castsInThread) {
+          expectDefined(cast.hash);
+        }
+        expect(castsInThread.length).to.be.greaterThan(10);
+      });
+    });
+
+    describe("#fetchCastsForUser", function () {
+      it("can fetch at least one cast", async function () {
+        let foundCast = false;
+        // eslint-disable-next-line no-unreachable-loop
+        for await (const cast of client.fetchCastsForUser({
+          fid: userGaviFid,
+        })) {
+          foundCast = true;
+          expect(cast.author.fid).to.eq(userGaviFid.toString());
+          expectDefined(cast.timestamp);
+          break;
+        }
+        expect(foundCast).to.be.true;
+      });
+
+      it("can fetch multiple pages worth of casts", async function () {
+        let castCount = 0;
+        const castCountBiggerThanPageSize = 150;
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        for await (const _ of client.fetchCastsForUser({ fid: userDwrFid })) {
```

Comment:
I wouldn't rely on the default value here
```suggestion
        const castCountBiggerThanPageSize = 150;
        // eslint-disable-next-line @typescript-eslint/no-unused-vars
        for await (const _ of client.fetchCastsForUser({ fid: userDwrFid }, {pageSize: 100)) {
```

Reply:
👍